fastlane_bot/events/utils.py

```diff
@@ -591,14 +593,17 @@ def get_config(
         The Tenderly fork ID, by default None
     self_fund : bool
         The bot will default to using flashloans if False, otherwise it will attempt to use funds from the wallet.
+    rpc_url : str, optional
+        The RPC URL to use, by default None
     Returns
     -------
     Config
         The config object.
 
     """
     default_min_profit_gas_token = Decimal(default_min_profit_gas_token)
-
+    if rpc_url:
+        rpc_url = NetworkBase.get_rpc_url(blockchain, tenderly_fork_id, rpc_url)
```

Comment:
Pointless, because at present, if `rpc_url` evaluates to a true result (i.e., it is a string of length larger than zero), then `get_rpc_url` simply returns this value as is. If anything, it should be called `if not rpc_url`.

Reply:
see revised